packages/node/src/machine/xkeys.ts

```diff
@@ -10,6 +11,10 @@ import { fromExtendedKey, HDNode } from "ethers/utils/hdnode";
  * BIP-44 specifies that if the purpose is 44, then the format is "m / purpose' / cointype' / account' / change / index"
  */
 
+export function computeRandomExtendedKey(): string {
```

Comment:
`computeRandomXpriv`

Reply:
I prefer ExtendedKey, or NonNeuteredExtendedKey

Reply:
Alright well these are opposing styles and we need to pick one. I'm leaning towards `xprv` simply because it's much shorter than `NonNeuteredExtendedKey`

Reply:
to be clearer: "`createRandomExtendedKey`" is ambiguous because it doesn't specify whether the output will begin with `xpub` or `xprv` and it's important for the caller of this function to know. `createNonNeuteredExtendedKey` is not ambiguous, hence acceptable to me on that point. however the phrase "non-neutered extended key" is not commonly used AFAICT

I think using the terminology from https://bitcoin.org/en/wallets-guide#hierarchical-deterministic-key-creation is best, it has all the terms we need and doesn't use the word "neuter"

Reply:
Inspired by the referenced guide, "extended {private,public} key" is used as of this PR.